weave-js/src/components/PagePanelComponents/Home/Browse3/datasets/schemaUtils.test.ts

```diff
@@ -0,0 +1,116 @@
+import {flattenObject, inferSchema, inferType} from './schemaUtils';
```

Comment:
note: Let's verify these tests run - i am not sure they are picked up automatically

Reply:
These getting picked up and run in the `WeaveJS Lint and Compile` CI step